packages/e2e/specs/overview/display-initial-os-version.e2e.ts

```diff
@@ -0,0 +1,26 @@
+import OverviewPage from "../../page-objects/overview.page"
+
+describe("Overview Page", () => {
+  describe("Have initial OS version", () => {
+    before(async () => {
+      // Waiting for device connected through USB
+      await browser.executeAsync((done) => {
+        setTimeout(done, 10000)
+      })
+    })
+
+    it("should have valid OS version", async () => {
+      const VERSION_REGEXP = /[0-9].[0-9].[0-9]/
```

Comment:
🟡
100.0.0 - fails
1.0.02 - fails

What do you think to use `semver.valid` from `semver` lib? 🤔

Reply:
https://github.com/mudita/mudita-center/pull/611/commits/b7e381a9b47bf9ffbf1f014e9fe4df4194fa8563